casper/src/main/scala/coop/rchain/casper/genesis/contracts/ConfigRegistry.scala

```diff
@@ -0,0 +1,12 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.models.NormalizerEnv
+import coop.rchain.rholang.build.CompiledRholangTemplate
+import coop.rchain.shared.Base16
+
+final case class ConfigRegistry(systemContractPubKey: String)
```

Comment:
Why this is not just `Registry`?

Reply:
@nzpr beacuse `Registry` module already exists in `rchain/rholang/src/main/scala/coop/rchain/rholang/interpreter/Registry/Registry.scala`

Reply:
its fine, this is the file under `casper/src/main/scala/coop/rchain/casper/genesis/contracts/` path, so it is a contract. If one wants to use both `rchain/rholang/src/main/scala/coop/rchain/rholang/interpreter/Registry/Registry.scala` and this, he can do
`import rholang.interpreter.{Registry => InterpreterRegistry}`
`import casper.genesis.contracts.{Registry => RegistryCompiledTemplate}`

At least you can use name `CompiledRegistryTemplate`, as the class extends `CompiledRholangTemplate`, but I think just `Registry` is fine

Reply:
@nzpr ok